### DISCUSSION TITLE
How can I only update the html files without updating the whole app?

### DISCUSSION THREAD

**Discussion by Almo7aya** [tauri-apps/tauri#6948] [Q&A]
We have a web application that we are using inside tauri, we rapidly deploy new features and fixes for our web app and don't want our users to update the whole tauri app ~120MB everytime we deploy something new, so is there a way to only update the html files without updating the whole app?

**Comment by simonhyll** [comment]
I'd say you have 2 options. You can either
A) Switch to using your app as a dynamic library https://github.com/tauri-apps/tauri/tree/dev/examples/tauri-dynamic-lib
B) Load the html from files on the disk

**Reply by FabianLars** [reply]
3) have your tauri app serve the hosted remote website, of course this makes your app require an internet connection to work...
